web-api/src/storage/elastic.rs

```diff
@@ -442,10 +456,11 @@ pub(crate) enum IndexUpdateMethod {
 }
 
 #[derive(Debug, Serialize)]
-struct IngestedDocument<'a> {
+struct EsDocument<'a> {
```

Comment:
:art: this is the only `Document` type comming from the `elastic` module, so we could just skip the prefix
```suggestion
struct Document<'a> {
```